exonum-java-binding/core/rust/exonum-java/TUTORIAL.md

```diff
@@ -19,20 +19,6 @@ in the [user guide](https://exonum.com/doc/version/0.11/get-started/java-binding
 
 ### Step 1. Configure Environment
 
-#### `LD_LIBRARY_PATH`
```

Comment:
We also need to mention that set `JAVA_HOME` will change the JVM to use. Perhaps in a different, section

Reply:
Just thinking if it's worth mentioning. Is there any other reason to set `JAVA_HOME` except explicit pointing to a different JVM? 🤔

Reply:
Probably in the installation/configuration instructions. I think it is, because it allows to configure the VM to use.

Reply:
https://github.com/exonum/exonum-doc/pull/331

Reply:
also https://jira.bf.local/browse/ECR-3048